src/components/simpleareachart.js

```diff
@@ -104,7 +104,7 @@ export default function SimpleAreaChart({ emissionsData, title }) {
       <ResponsiveContainer className="simplearea-cont">
         <AreaChart
           width={800}
-          height={400}
+          height={200}
```

Comment:
It doesn't have to be completely proportional, but should we also make the graph less wide? Otherwise it looks a little silly and makes the vertical changes hard to see:

![Screenshot from 2023-01-15 22-06-16](https://user-images.githubusercontent.com/3187531/212596537-f6aab3c6-
bb73-45ca-80c8-691c353e75fc.png)

Reply:
i think its ok. especially when looking at the chart in mobile

Reply:
btw changing this had no impact on the chart height. for some reason it was only changed in the `simplearea-cont` css class